src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+}
+
+.stars__star {
+  background-image: url(Star.png);
```

Comment:
looks like you've used not images from this task. You have images at `images` folder